internal/test/components/components.gen.go

```diff
@@ -171,7 +170,7 @@ func (a ParamsWithAddPropsParams_P1) MarshalJSON() ([]byte, error) {
 	for fieldName, field := range a.AdditionalProperties {
 		object[fieldName], err = json.Marshal(field)
 		if err != nil {
-			return nil, errors.Wrap(err, fmt.Sprintf("error marshaling '%s'", fieldName))
+			return nil, fmt.Errorf("error marshaling '%s'", fieldName)
```

Comment:
Throughout this change, you're losing the wrapped error.  This would be equivalent to the old error:
```suggestion
			return nil, fmt.Errorf("error marshaling '%s': %w", fieldName, err)
```

Reply:
got it, thanks.